spec/controllers/hq/sponsors_controller_spec.rb

```diff
@@ -1,16 +1,19 @@
 require 'rails_helper'
+require 'will_paginate/array'
 
 RSpec.describe Hq::SponsorsController, type: :controller do
   before :each do
     sign_in instance_double(AdminUser)
     @sponsor = FactoryGirl.build_stubbed :sponsor
+    @sponsors = (1..5).map {FactoryGirl.build_stubbed :sponsor}
     @sponsor_build = FactoryGirl.build :sponsor
     @sponsor_new = Sponsor.new
   end
```

Comment:
Permission to change all these instance variables into `let` otherwise all will be loaded even when not necessary.